src/semidiscretization/semidiscretization_hyperbolic_parabolic.jl

```diff
@@ -0,0 +1,256 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
+"""
+    SemidiscretizationHyperbolicParabolic
+
+A struct containing everything needed to describe a spatial semidiscretization
+of a mixed hyperbolic-parabolic conservation law.
+"""
+struct SemidiscretizationHyperbolicParabolic{Mesh, Equations, ParabolicEquations, InitialCondition,
+                                             BoundaryConditions, ParabolicBoundaryConditions,
+                                             SourceTerms, Solver, Cache, ParabolicCache} <: AbstractSemidiscretization
```

Comment:
```suggestion
struct SemidiscretizationHyperbolicParabolic{Mesh, Equations, EquationsParabolic, InitialCondition,
                                             BoundaryConditions, BoundaryConditionsParabolic,
                                             SourceTerms, Solver, Cache, CacheParabolic} <: AbstractSemidiscretization
```
For consistency? Should be changed in other places in this file as well then. Also, there are further variables that would need to be changed, e.g., `parablic_boundary_conditions`, `hypberolic_equations` etc. Maybe just leave it for later and just add it to #1147 for now?

Reply:
Whoops, yes! This is an issue with search-replace...

If it's OK, I'd prefer to fix it here to get it over with.